Client/Frontend/Home/Wallpapers/v2/UI/WallpaperSelectorViewModel.swift

```diff
@@ -0,0 +1,116 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+class WallpaperSelectorViewModel {
+
+    enum WallpaperSelectorLayout: Equatable {
+        case compact
+        case regular
+
+        // The maximum number of items to display in the whole section
+        var maxItemsToDisplay: Int {
```

Comment:
nits for this file: return on the same line as the case as it's small and one line. Maybe. Maybe not. Up to you.